src/simulator/halo_simulator.py

```diff
@@ -243,40 +243,54 @@ def simulated_venn_diagram_reach_by_spend(
     def _form_venn_diagram_regions(
         self, spends: List[float], max_frequency: int = 1
     ) -> Dict[int, List]:
-        """Form Venn diagram regions that contain k+ reaches
+        """Form primitive Venn diagram regions that contain k+ reaches
 
-        For each subset of publishers, computes k+ reaches for those users
-        who are reached by the publishers in that subset.
+        For each subset in the powerset of publishers, computes k+ reaches for
+        those users who are reached by the publishers in that subset.
 
         Args:
-            spends:  The hypothetical spend vector, equal in length to
-              the number of publishers.  spends[i] is the amount that is
-              spent with publisher i. Note that publishers with 0 spends will
-              not be included in the Venn diagram reach.
+            spends:  The hypothetical spend vector, equal in length to the
+              number of publishers.  spends[i] is the amount that is spent with
+              publisher i. Note that the publishers with 0 spends, inactive
+              publishers, will not be included in the Venn diagram regions.
             max_frequency:  The maximum frequency for which to report reach.
         Returns:
-            regions:  A dictionary in which each key are the binary
+            regions:  A dictionary in which each key is the binary
               representations of each primitive region of the Venn diagram, and
```

Comment:
Change to 'representation of a primitive region...'

Reply:
Do you mean "regions:  A **representation of a primitive region** in which each key is the binary..."?